pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -60,13 +60,24 @@ func newCephStatusChecker(context *clusterd.Context, resourceName string, cephUs
 	}
 
 	// allow overriding the check interval with an env var on the operator
-	checkInterval := os.Getenv("ROOK_CEPH_STATUS_CHECK_INTERVAL")
-	if checkInterval != "" {
-		if duration, err := time.ParseDuration(checkInterval); err == nil {
-			logger.Infof("ceph status check interval is %s", checkInterval)
-			c.interval = duration
-		}
+	// Keep the existing behavior
+	var checkInterval string
+	checkIntervalCRSetting := healthCheck.DaemonHealth.Status.Interval
+	checkIntervalEnv := os.Getenv("ROOK_CEPH_STATUS_CHECK_INTERVAL")
+	if checkIntervalEnv != "" {
+		checkInterval = checkIntervalEnv
+	}
+
+	if checkIntervalCRSetting != "" && checkIntervalEnv == "" {
+		checkInterval = checkIntervalCRSetting
 	}
+
+	// Set duration
+	if duration, err := time.ParseDuration(checkInterval); err == nil {
```

Comment:
`checkInterval` might still not be set in the CR or operator env var, right? Where is the default specified in that case?

Reply:
By default in the struct above with `defaultStatusCheckInterval`